src/core/util/log.cc

```diff
@@ -128,3 +129,7 @@ void gpr_log_verbosity_init(void) {
   }
 #endif  // GRPC_VERBOSITY_MACRO
 }
+
+GPRAPI void grpc_absl_log_init(void) {
```

Comment:
@sourabhsinghs 

Please note that we'll need to sync with the core team if we're going to be adding a Core API.

I think it might be more reasonable to instead call this abls function directly from our Cython code.

Reply:
Yes, calling it directly in Cython is probably better, you can include absl like this:
```
cdef extern from "absl/log/initialize.h" namespace "absl" nogil
```

For bazel, we probably need add `@com_google_absl//absl/log:initialize` to deps.

Reply:
have updated to call InitializeLog from Cython itself as suggested